cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -59,7 +59,7 @@ EOF
 done
 
 for workdir in $( find /tmp/deprovision -mindepth 1 -type d | shuf ); do
-  openshift-install --dir "${workdir}" --log-level debug destroy cluster
+  timeout 30m openshift-install --dir "${workdir}" --log-level debug destroy cluster
```

Comment:
This would exit the script.

Should we ignore the error and continue with the `for-loop`?

Reply:
I would say so, yes. The goal is to attempt to delete all clusters even if one fails. The `other` pruner should catch those that are not deleting here.

Reply:
@hongkailiu I think it's fine to abort with a failure, it's essentially the same like `openshift-install` exiting unsuccessfully, and we run the tool often enough so that the shuffler makes it clean the remaining clusters eventually.

Reply:
ack. Updated.

Reply:
Removed again.

We want to see the alerts of failing jobs.